From: Stefano Rivera <stefanor@debian.org>
Date: Mon, 10 Mar 2025 12:55:36 -0400
Subject: Another missing #include

Can't find the upstream equivalent to this, it must be buried elsewhere
in the include stack.
---
 test/cpp/end2end/xds/xds_server.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/cpp/end2end/xds/xds_server.h b/test/cpp/end2end/xds/xds_server.h
index 6fe7167..db7fd97 100644
--- a/test/cpp/end2end/xds/xds_server.h
+++ b/test/cpp/end2end/xds/xds_server.h
@@ -27,6 +27,7 @@
 #include <gtest/gtest.h>
 
 #include "absl/types/optional.h"
+#include "absl/strings/numbers.h"
 
 #include <grpc/support/log.h>
 
